.circleci/config.yml

```diff
@@ -145,21 +137,13 @@ jobs:
       # restore_cache step above can override the go-filecoin code with cached code
       - git_fetch_all_tags
       - checkout
-      - restore_cache:
```

Comment:
BLOCKING: If you eliminate this `restore_cache`, you will _always_ attempt to download (and, failing that, generate) Groth parameters and verifying keys during the `"deps"` command.